annotation-protocol/requirements.txt

```diff
@@ -1 +1 @@
-wptserve
+wptserve==1.4.0
```

Comment:
This seems totally bogus, given wptserve/wptrunner on PyPI is essentially nonsense.